index.js

```diff
@@ -0,0 +1,14 @@
+
+// conservative limit for now
+const TENANT_ID_REGEX = new RegExp(/^[a-zA-Z][a-zA-Z-_]{0,98}[a-zA-Z]$/);
+const UUIDV4_REGEX = new RegExp(/^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i); 
```

Comment:
Would it be better to encourage the validation function provided by the uuid library instead of using a regex for this?
https://github.com/uuidjs/uuid#uuidvalidatestr

Reply:
ill resolve this, as i stole the regex from this repo :)